Parainfo (#1904)
w3f/polkadot-wiki: docs/learn-parachains.md

and can communicate with other parachains through
[XCMP](https://wiki.polkadot.network/docs/en/learn-crosschain).

Parachains are maintained by a network maintainer known as a
[collator](https://wiki.polkadot.network/docs/en/learn-collator). The role of the collator node is
to maintain a full-node of the parachain, retain all necessary information of the parachain, and
produce new block candidates to pass to the Relay Chain validators for verification and inclusion in
the shared state of Polkadot. The incentivization of a collator node is an implementation detail of
the parachain. They are not required to be staked on the Relay Chain or own DOT tokens unless
stipulated to do so by the parachain implementation.

The Polkadot Host (PH) requires that the state transitions performed on parachains to be specified
as a Wasm executable. Proofs of new state transitions that occur on a parachain must be validated
against the registered state transition function (STF) that is stored on the Relay Chain by the
validators before {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} acknowledges a

> state transition has occurred on a parachain. The only constraint to the logic that a parachain is
> allowed to implement is that it must be verifiable by the Relay Chain validators. Verification most

Reviewer: ```suggestion
state transition has occurred on a parachain. The key constraint regarding the logic of a parachain is
that it must be verifiable by the Relay Chain validators. Verification most
```
I believe there are other constraints here, e.g. in terms of relying on relay chain finalization,  block production time, etc.

commonly takes the form of a bundled proof of a state transition known as a Proof-of-Verification
(PoV) block, which is submitted to the validators from one or more of the parachain collators to be
checked.

## Parachain Economies

Parachains may have their own economies with their own native tokens. Schemes such as Proof-of-Stake
are usually used to select the validator set in order to handle validation and finalization;
parachains will not be required to do either of those things. However, since

> {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} is general over what the parachain

Reviewer: ```suggestion
{{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} is not overly particular about what the parachain
```
